e2e/cypress/integration/system_console/revoke_all_sessions.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+const buttonTheme = {backgroundColor: 'rgba(0, 0, 0, 0.7)', color: 'rgb(255, 255, 255)'};
+
+describe('System Console', () => {
+    it('SC17020 - Revoke All Sessions from System Console', () => {
+        // # Login as System Admin
+        cy.apiLogin('sysadmin').as('sysadmin');
+
+        cy.visit('/admin_console/user_management/users');
+
+        // * Verify the presence of Revoke All Sessions button
+        cy.get('#revoke-all-users').should('be.visible').and('have.css', 'background-color', buttonTheme.backgroundColor).
+            and('have.css', 'color', buttonTheme.color);
+
+        cy.get('#revoke-all-users').click();
```

Comment:
What's the behavior of this with a full user list? Can the button be hidden? Do we need to scroll it into view first?

Reply:
This button is displayed always irrespective of the number of users. 
![image](https://user-images.githubusercontent.com/1429138/61548742-fd55ec00-aa6b-11e9-86fb-af6d298aa0dd.png)